.github/workflows/sycl_update_gpu_driver.yml

```
@@ -0,0 +1,58 @@
+name: Update GPU driver
+
+on:
+  schedule:
+    - cron:  '0 3 * * 2'
+  workflow_dispatch:
+
+jobs:
+  update_driver_linux:
+    runs-on: ubuntu-latest
+    if: github.repository == 'intel/llvm'
+    steps:
+    - uses: actions/checkout@v2
+    - name: Update dependencies file
+      run: |
+        version="$(python3 devops/scripts/update_drivers.py linux)"
+        echo 'NEW_DRIVER_VERSION='$version >> $GITHUB_ENV
+    - name: Create Pull Request
+      env:
+        BRANCH: ci/update_gpu_driver-linux-${{ env.NEW_DRIVER_VERSION }}
+        LLVMBOT_TOKEN: ${{ secrets.LLVM_MAIN_SYNC_BBSYCL_TOKEN }}
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      run: |
+        cd $GITHUB_WORKSPACE
+        # Set fake identity to fulfil git requirements
+        git config --global user.name "iclsrc"
+        git config --global user.email "ia.compiler.tools.git@intel.com"
+        git checkout -B $BRANCH
+        git add -u
+        git commit -m "[GHA] Uplift GPU RT version for Linux CI" || exit 0   # exit if commit is empty
+        git push https://$LLVMBOT_TOKEN@github.com/${{ github.repository }} ${BRANCH}
+        gh pr create --title "[GHA] Uplift GPU RT version for Linux CI" --body "Uplift GPU RT version for Linux to $NEW_DRIVER_VERSION" --assignee @intel/dpcpp-devops-reviewers
+
+  update_driver_linux_staging:
+    runs-on: ubuntu-latest
+    if: github.repository == 'intel/llvm'
+    steps:
+    - uses: actions/checkout@v2
+    - name: Update dependencies file
+      run: |
+        version="$(python3 devops/scripts/update_drivers.py linux_staging)"
+        echo 'NEW_DRIVER_VERSION='$version >> $GITHUB_ENV
+    - name: Create Pull Request
+      env:
+        BRANCH: ci/update_gpu_driver-linux_staging-${{ env.NEW_DRIVER_VERSION }}
+        LLVMBOT_TOKEN: ${{ secrets.LLVM_MAIN_SYNC_BBSYCL_TOKEN }}
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      run: |
+        cd $GITHUB_WORKSPACE
+        # Set fake identity to fulfil git requirements
+        git config --global user.name "iclsrc"
+        git config --global user.email "ia.compiler.tools.git@intel.com"
```

```suggestion
        git config --local user.email "actions@github.com"
        git config --local user.name "GitHub Actions"
```
Please, apply to the same code above.

---

done. For testing on my personal fork, these need to be valid values (cperkinsintel, etc).  I'm fairly sure that on our main repo, the GITHUB_TOKEN will serve to authorize and these values can be set however we want (though they must be set). But presently it is untested, at least by me.